hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/regression/SteadyStateThrottlingCheck.java

```diff
@@ -231,7 +225,7 @@ final HapiSpec checkCustomNetworkTps(
                 }));
     }
 
-    @HapiTest
+    @BddMethodIsNotATest
```

Comment:
This still doesnt work ?

Reply:
This method is a factory for `HapiSpec`'s, but not itself a directly runnable `HapiSpec`.